proxy-exports/icons/index.js

```diff
@@ -0,0 +1,93 @@
+export { AddBoldIcon } from '../ui-kit.esm';
```

Comment:
Is this list generated or not?

Reply:
We created it manually in hope of all future consumers importing from `ui-kit.esm` instead.

We hope to be able to remove all these proxy-exports soon. So it didn't feel reasonable to write a script for it if we're going to remove it soon anyways. We want any new icons to be consumed as a named export from the main file.

Reply:
Sure, I also improved the docs a bit to mention this: https://github.com/commercetools/ui-kit/pull/34/commits/4985d0be982bb3afef8ccad751bcb0083e5fdb3b